airflow/dags/model_retrain_dag.py

```
@@ -0,0 +1,399 @@
+import xgboost as xgb
+import pandas as pd
+from sklearn.metrics import (
+    r2_score,
+    mean_squared_error,
+    mean_absolute_error,
+    mean_absolute_percentage_error,
+)
+import numpy as np
+import joblib
+import os
+import psycopg2
```

Ruff (0.11.9)

`psycopg2` imported but unused

Remove unused import: `psycopg2`

(F401)